addons/mail/static/tests/discuss/call/call.test.js

```diff
@@ -690,3 +691,58 @@ test("shows warning on infinite mirror effect (screen-sharing then fullscreen)",
     await hover(queryFirst("button:contains('Stream paused')"));
     await contains("button:contains('Resume stream') i.fa-play-circle-o");
 });
+
+test("join call with previous call settings (audio only)", async () => {
```

Comment:
Name test in a more straight-forward way of functional purpose.
Current name is too broad in understanding what is useful, forcing us to read the test to understand to get what's relevant.

With a better name, reading test content is optional and just the name is enough to get the intent, saving lots of time and making test more maintainable.

Suggestion: _"single 'join' (without camera) button when last call was audio-only"_